masterlist.yaml

```
@@ -23577,6 +23577,108 @@ plugins:
       - <<: *alreadyInOrFixedByX
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp")'
+    
+    ### Dawn of Skyrim - Modern Patches ###
+  - name: 'DoS_MP_Whiterun.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/65467/' ]
+    msg:
+      - <<: *patchIncluded
+        subs: [ '***Interesting NPCs***' ]
+        condition: 'active("3DNPC.esp") and not active("DoS_MP_Whiterun_3DNPC.esp")'
+      - <<: *patchIncluded
+        subs: [ '***Capital Whiterun Expansion***' ]
+        condition: 'active("SurWR.esp") and not active("DoS_MP_Whiterun_CWE.esp")'
+      - <<: *patchIncluded
+        subs: [ '***Dynamic Things Alternative***' ]
+        condition: 'active("Dynamic Things Alternative.esp") and not active("DoS_MP_Whiterun_DynamicThingsAlternative.esp")'
+      - <<: *patchIncluded
+        subs: [ '***Fishing***' ]
+        condition: 'active("ccBGSSSE001-Fish.esm") and not active("DoS_MP_Whiterun_Fishing.esp")'
+      - <<: *patchIncluded
+        subs: [ '***JKs Skyrim***' ]
+        condition: 'active("JKs Skyrim.esp") and not active("DoS_MP_Whiterun_JK.esp")'
+      - <<: *patchIncluded
+        subs: [ '***Legacy of the Dragonborn***' ]
+        condition: 'active("LegacyoftheDragonborn.esm") and not active("DoS_MP_Whiterun_LOTD.esp") and not active("DoS_MP_Whiterun_CWE_LOTD.esp") and not active("DoS_MP_Whiterun_JK_LOTD.esp") and not active("DoS_MP_Whiterun_JK_CWE_LOTD.esp")'
```

Some of the conditions can be simplified with regexes:

```suggestion
        condition: 'active("LegacyoftheDragonborn.esm") and not active("DoS_MP_Whiterun_(JK_)?(CWE_)?LOTD\.esp")'
```